README.md

```diff
@@ -51,3 +51,10 @@ Features
 Tutorials
 --------
 - [GPS Tracks Animation mit "GPX Animator" (Marcus Bersheim)](https://www.youtube.com/watch?v=AtcBVrbB6bg) :de:
+
+
+Credits
+--------
+Icons included in application and their source:
+
+* <div>Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
```

Comment:
This is a Markdown file, please write this line in Markdown syntax.
https://guides.github.com/features/mastering-markdown/#syntax
Or take a look at line 53. There is a link, too.